stacker_blueprints/firehose/base.py

```diff
@@ -233,27 +229,60 @@ class Firehose(Blueprint):
                            "Default: true",
             "default": True,
         },
-        "KeyUseArns": {
+        "KeyUserArns": {
             "type": list,
-            "description": "A list of ARNs that need access to the KMS key "
-                           "created with EncryptS3Bucket",
-            "default": [],
+            "default": []
         },
         "KeyAdminArns": {
             "type": list,
-            "description": "A list of ARNs that need to admin the KMS key "
-                           "created with EncryptS3Bucket",
             "default": [],
+        },
+        "SizeInMBs": {
+            "type": int,
+            "description": "Size in MBs for buffering hints"
```

Comment:
Maybe: `Size in MBs for buffering hints for the Firehose Delivery Stream`?  Just to be more explicit.  Some of the following could be a little more explicit too.